packages/grafana-ui/src/components/Icon/iconBundle.ts

```diff
@@ -165,7 +165,7 @@ import u1121 from '../../../../../public/img/icons/unicons/wrap-text.svg';
 import u1135 from '../../../../../public/img/icons/unicons/x.svg';
 
 export let cacheInitialized = false;
-export let iconRoot = 'public/img/icons';
+export let iconRoot = 'public/img/icons/';
```

Comment:
this change is vital. It fixes a current problem where icons not on the list of pre-cached icons won't load because the backup network request fails due to a broken url that misses this trailing slash.